podcast/alisha-johnson-wilder.html

```diff
@@ -0,0 +1,289 @@
+              <h3>LISTEN NOW</h3>
+              <div class="podcast-player">
+                <img
+                  src="../assets/images/podcast/friends-with-mental-health-benefits.jpeg"
+                  alt="friends with mental health benefits"
+                  class="thumb"
+                />
+                <div>
+                  <div class="gra6">July 7, 2022 . 45 MIN</div>
+                  <div>Faye McCray — Walking In Your Purpose</div>
+                  <div class="gray">Friends with Mental Health Benefits</div>
+                  <div class="btns">
+                    <button class="play-btn">
+                      <img
+                        src="../assets//images/icons/play.svg"
+                        alt="play icon"
+                        class="icon"
+                      /><span>Play</span>
+                    </button>
+                    <button class="see-more-btn">
+                      <span>See More</span>
+                      <img
+                        src="../assets//images/icons/arrow-right-top.svg"
+                        alt="arrow right top icon"
+                        class="icon"
+                      />
+                    </button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="detail">
+            <h2>Episode Details</h2>
+            <p>
+              One of <a href="#">Faye McCray’s</a> biggest focuses as a mom:
+              Giving her sons an example of what happiness looks like. In this
+              episode, the editor-in-chief of <a href="#">Psych Central</a> and
+              former attorney talks with Marah about pursuing her second act as
+              a journalist, building a more inclusive conversation around mental
+              health, and creating space as a leader for the days when people
+              aren’t OK.
+            </p>
+            <p>
+              Friends with Mental Health Benefits is brought to you by Shine at
+              Work, an employee wellbeing program rooted in culturally competent
+              care from the team behind the award-winning Shine app. Learn more
+              about Shine at Work.
+            </p>
+          </div>
+        </div>
+      </section>
+      <img
+        src="../assets/images/light-purple-bottom-wave.svg"
+        alt="light purple bottom wave"
+        class="light-purpple-bottom-wave-bg"
+      />
+      <section class="podcast-speakers-block">
+        <h2 class="heading-xl">Episodes</h2>
+        <ul class="podcast-speaker-list">
+          <li>
+            <div class="podcast-speaker-card">
+              <img
+                src="../assets/images/podcast/speakers/reshma.png"
+                alt="resmha"
+                class="speaker-image"
+              />
+              <h3 class="heading-s">EPISODE 1</h3>
+              <h4 class="heading-m">WORKPLACES ARE BURNING OUT WORKING MOMS</h4>
```

Comment:
@chemagaton24, based on the design, 'Episode 1' doesn’t appear to be a heading and could work as an eyebrow. We could use a regular paragraph or span tag to replace the `h4` with an `h3`.

Reply:
Please apply to the rest of the PR where applicable.